packages/base-driver/lib/basedriver/core.ts

```diff
@@ -99,6 +101,10 @@ export class DriverCore<const C extends Constraints, Settings extends StringReco
 
   protocol?: Protocol;
 
+  bidiEventSubs: Record<string, string[]>;
```

Comment:
the abbreviation `Subs` might be confusing. How about `Subscriptions`?

Reply:
i think this abbreviation is fairly global. I'm open to changing it. Subscriptions is so long though :-/

Reply:
how about `bidiEventSubscriptions`? `Subs` might be confused to `sub procedures`